etcdmgmt/etcd-mgmt_test.go

```diff
@@ -20,10 +48,11 @@ func TestStartETCD(t *testing.T) {
 	etcdCtx, err := StartETCD()
 	tests.Assert(t, err == nil)
 	etcdCtx.Kill()
+	etcdCtx.Wait()
 }
 
 func TestWriteETCDPidFile(t *testing.T) {
-	cmd := exec.Command("etcd")
+	cmd := formETCDCommand()
```

Comment:
if that is the case then no need to check error here. this comment is valid only if we are getting error.